.github/workflows/int-and-connected-test-run.yml

```diff
@@ -161,47 +161,19 @@ jobs:
           GIT_HASH=$(git rev-parse --short HEAD)
           echo "GIT_HASH=${GIT_HASH}" >> $GITHUB_OUTPUT
           echo "Latest git hash in branch is ${GIT_HASH}"
-  build-and-push-image:
-    runs-on: ubuntu-latest
-    needs:
-      - git_hash
-    steps:
-      - name: Checkout tagged branch of jade-data-repo
-        uses: actions/checkout@v4
-        with:
-          fetch-depth: 0
-          persist-credentials: false
-      - name: Set up JDK 17
-        uses: actions/setup-java@v4
-        with:
-          distribution: 'temurin'
-          java-version: '17'
-          cache: 'gradle'
-      - name: 'Build and push image GAR'
-        run: |
-          # extract service account credentials
-          base64 --decode <<< ${{ secrets.SA_B64_CREDENTIALS }} > ${GOOGLE_APPLICATION_CREDENTIALS}
-          jq -r .private_key ${GOOGLE_APPLICATION_CREDENTIALS} > ${GOOGLE_SA_CERT}
-          chmod 644 ${GOOGLE_SA_CERT}
-          # Build, tag and push the image
-          ./gradlew jib
-        env:
-          GCR_TAG: ${{ needs.git_hash.outputs.version }}
-          GOOGLE_APPLICATION_CREDENTIALS: jade-dev-account.json
-          GOOGLE_SA_CERT: jade-dev-account.pem
   report-to-sherlock:
```

Comment:
Can this be removed too? It is also being done in `build.yaml` which is running on all branches now (as I understand it).

Reply:
Yes, thanks!